docs/console.html

```diff
@@ -0,0 +1,804 @@
+inspect($_)
+</code></pre>
+        </figure>
+    </section>
+
+    <section>
+        <h3 id="accessing-recently-selected-elements-and-objects">Accessing recently selected elements and objects</h3>
+        <p>
+            The console has access to the last five elements or objects that were selected.
+            As you select an element in the Elements panel or an object in the Profiles panel, that is pushed onto the history stack.
+            The history stack is accessed by using <code><a href="commandline-api#0-4">$<var>x</var></a></code> where <var>x</var> is the number of the item you wish to access.
+            Remember computers begin counting from 0; this means the first item is <code>$0</code> and the last item is <code>$4</code>.
+        </p>
+    </section>
+
+    <section>
+        <h3 id="monitoring-events">Monitoring events</h3>
+        <p>
+            The <code><a href="commandline-api#monitoreventsobject-events">monitorEvents()</a></code> method instructs the DevTools to log information on the specified targets.
+            The first parameter is the object to monitor.
+            To specify the events to listen to you may pass either a string or an array of strings as the second parameter.
+        </p>
+        <figure>
+            <figcaption>Listen to click events on the body of the page.</figcaption>
+            <pre class="prettify"><code class="language-javascript">
+monitorEvents(document.body, "click");
+</code></pre>
+        </figure>
+        <p>
+            If a supported <i>event type</i> that the DevTools maps to a set of standard event names is provided then it will listen to all of the events for that type.
+            A full list of <i>event types</i> can be found in the <a href="commandline-api#monitoreventsobject-events">Command Line API</a>.
+        </p>
+        <p>
+            To stop monitoring events you call the <code>unmonitorEvents()</code> method and give it the object to stop monitoring.
+        </p>
+        <figure>
+            <figcaption>
+                Stop listening to events on the <var>body</var> object.
+            </figcaption>
+            <pre class="prettify"><code class="language-javascript">
+unmonitorEvents(document.body);
+</code></pre>
+        </figure>
+    </section>
+
+    <section>
+        <h3 id="controlling-the-cpu-profiler">Controlling the CPU Profiler</h3>
+        <p>
+            JavaScript CPU profiles can be run from the command line as well.
+            This is achieved with the <code>profile()</code> function.
+            You may pass in a string to name the profile as well.
+            To stop a profile call <code>profileEnd()</code> in the same way you called the initializer.
+        </p>
+        <figure>
+            <figure>
+            <figcaption>
+                Create a new profile without providing a name.
+            </figcaption>
+            <pre class="prettyprint"><code class="language-javascript">
+profile()
```

Comment:
these are in the console API and DEFINITELY more useful there instead of cmd line api.
i'd rather document profile/End up there. 

do we have the full console API covered above?

![image](https://cloud.githubusercontent.com/assets/39191/6384378/add938be-bd10-11e4-9ec2-cdcf5c7ad4bf.png)